account_financial_report_qweb/report/abstract_report_xlsx.py

```diff
@@ -3,6 +3,8 @@
 # Copyright 2016 Camptocamp SA
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
 
+from cStringIO import StringIO
+import xlsxwriter
```

Comment:
But then the previous PR was incorrect. We can't mix XLSX generation with PDF generation. That should be optional through a module  account_financial_report_qweb_xlsx